tests/cloner_test.go

```diff
@@ -75,6 +76,219 @@ var _ = Describe("[rfe_id:1277][crit:high][vendor:cnv-qe@redhat.com][level:compo
 	})
 })
 
+var _ = Describe("Validate creating multiple clones of same source Data Volume", func() {
+	f := framework.NewFrameworkOrDie(namespacePrefix)
+	tinyCoreIsoURL := fmt.Sprintf(utils.TinyCoreIsoURL, f.CdiInstallNs)
+
+	var (
+		sourceDv, targetDv1, targetDv2, targetDv3 *cdiv1.DataVolume
+		err                                       error
+	)
+
+	AfterEach(func() {
+		if sourceDv != nil {
+			By("Cleaning up source DV")
+			err = utils.DeleteDataVolume(f.CdiClient, f.Namespace.Name, sourceDv.Name)
+			Expect(err).ToNot(HaveOccurred())
+		}
+		if targetDv1 != nil {
+			By("Cleaning up target1 DV")
+			err = utils.DeleteDataVolume(f.CdiClient, f.Namespace.Name, targetDv1.Name)
+			Expect(err).ToNot(HaveOccurred())
+		}
+		if targetDv2 != nil {
+			By("Cleaning up target2 DV")
+			err = utils.DeleteDataVolume(f.CdiClient, f.Namespace.Name, targetDv2.Name)
+			Expect(err).ToNot(HaveOccurred())
+		}
+		if targetDv3 != nil {
+			By("Cleaning up target3 DV")
+			err = utils.DeleteDataVolume(f.CdiClient, f.Namespace.Name, targetDv3.Name)
+			Expect(err).ToNot(HaveOccurred())
+		}
+	})
+
+	It("Should allow multiple clones from a single source datavolume", func() {
```

Comment:
Please add the Polarion ID as follows:
It("[rfe_id:CNV-1277][case_id:CNV-1891][crit:High][vendor:cnv-qe@redhat.com][level:component] Should allow multiple clones from a single source datavolume", func() {

Reply:
Done